### PR TITLE
Remove mean_return helper and replace by evaluate_policy in SB3

### PR DESCRIPTION
**Pull request by ernestum**
Just a small PR to gradually reduce the number of helpers and utils that we have to maintain ourselves.

### CODE REVIEW

File: tests/algorithms/test_bc.py [hunk 1]
```diff
@@ -94,15 +94,14 @@ def test_train_end_cond_error(trainer: bc.BC):
...
-    sample_until = rollout.make_min_episodes(15)
-    novice_ret_mean = rollout.mean_return(trainer.policy, cartpole_venv, sample_until)
+    novice_ret_mean, _ = evaluation.evaluate_policy(trainer.policy, cartpole_venv, 15)
+
     trainer.train(
         n_epochs=1,
         on_epoch_end=lambda: print("epoch end"),
         on_batch_end=lambda: print("batch end"),
     )
-    trainer.train(n_batches=10)
```
Comment (AdamGleave) [inline tests/algorithms/test_bc.py:104]: why is this line being deleted?

**Review by AdamGleave — APPROVED** [review]
LGTM apart from deleting one line (premature removal of n_batches from test?)

File: tests/algorithms/test_bc.py (hunk 1 above)
Comment (ernestum) [reply tests/algorithms/test_bc.py:104]: That was an accident. I will add it back in.

**Review by AdamGleave — APPROVED** [review]
LGTM

**Pull request merged**